02-basic-cpp/README.md

```diff
@@ -1 +1,183 @@
-# Basic C++ Workshop
\ No newline at end of file
+# Basic C++ Workshop
+
+## Purpose
+
+In this workshop, you will learn how to write and edit basic C++ programs.
+If you are not able to attend the workshop, please complete it before onboarding starts as this is crucial to be able to work with your team.
+
+We will be available during the workshop time to help you if you have any problems related to the workshop exercises.
+If you have any other questions but are not able to ask during the workshop, don't hesitate to message us on Slack!
+
+## Preparation
+
+1. Go through this [guide](https://github.com/Hyp-ed/hyped-2022/wiki/CPP-Getting-Started) to set up a C++ development environment.
```

Comment:
Not sure if it's just me, but installing cmake did not work for me the way described. I had to instead do the following:
https://askubuntu.com/questions/1203635/installing-latest-cmake-on-ubuntu-18-04-3-lts-run-via-wsl-openssl-error

Reply:
Huh, that's interesting. Seems like more of a hassle than it should be?